src/main/resources/modules/allergic_rhinitis.json

```diff
@@ -360,5 +360,6 @@
     "Terminal": {
       "type": "Terminal"
     }
-  }
+  },
+  "gmfVersion": 1
```

Comment:
I know it makes no difference but, just for eye-balling convenience, it would be nice if this was the first property.

Reply:
This is kind of a pain to do with `JSON.stringify`